samples/subsys/usb/hid-keyboard/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Source common USB sample options used to initialize new experimental USB
+# device stack. You do not need to source this Kconfig file in your custom
+# application.
```

Comment:
Align this with the text in main that says this must not be used in applications.